dojo/tools/mobsf_scorecard/parser.py

```diff
@@ -0,0 +1,101 @@
+import json
+from datetime import datetime
+
+from dojo.models import Finding
+
+
+class MobSFScorecardParser:
+
+    def get_scan_types(self):
+        return ["MobSF Scorecard Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "MobSF Scorecard Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Export a JSON file using the API, api/v1/report_json."
+
+    def get_findings(self, filename, test):
+
+        tree = filename.read()
+
+        try:
+            data = json.loads(str(tree, "utf-8"))
+        except:
+            data = json.loads(tree)
+
+        if "timestamp" in data:
+            find_date = datetime.strptime(data["timestamp"], "%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
I have grown to be a little paranoid about relying on specific date formats. If a tool changes the date format out from under you, the parser is no longer functional.

My suggestion is use the `dateutil.parser` function ([example here](https://github.com/DefectDojo/django-DefectDojo/blob/fafe5c3cc5f1287963e4681979095b8b8064575f/dojo/tools/netsparker/parser.py#L39)), but a try/catch works in as well

Reply:
Sure thing, copy-pasted from your example, thanks for advice )